sample_templates/ec2.yml.j2

```diff
@@ -14,6 +14,59 @@ Metrics:
     InstanceId: "{{ instance.id }}"
   Options:
     {#- I'm assuming my tag names are safe to use as metric names here #}
-    Formatter: 'cloudwatch.%(Namespace)s.{{ instance.tags['Name'] }}.%(MetricName)s.%(statistic)s.%(Unit)s'
+    Formatter: 'collectd.{{ instance.tags['Name'] }}.%(MetricName)s.%(statistic)s.%(Unit)s'
```

Comment:
Yep your are correct. I thought i abstracted out all of these but I guess I missed it.